utils/text_formatting.jsx

```diff
@@ -64,6 +64,16 @@ export function formatText(text, inputOptions) {
     } else if (!('markdown' in options) || options.markdown) {
         // the markdown renderer will call doFormatText as necessary
         output = Markdown.format(output, options);
+        if (output.includes('class="markdown-inline-img"')) {
+            /*
+            ** remove p tag to allow other divs to be nested,
```

Comment:
Adding this was necessary as with the old implementation there was console errors about wrong DOM nodes as a `<div>` tag shouldn't be nested in a `<p>` tag. And if I don't replace the `<p>` with a `<div>` react produces a weird unique key error (though there are no lists I could find in all the relevant components in the rendering tree)